textproc/par/Portfile

```
@@ -18,26 +19,31 @@ master_sites        ${homepage}
 distname            [string totitle $name][string map {. {}} $version]
 
 checksums           rmd160  253070f319a7f3b3a6f0dad5797882efdbf597bc \
-                    sha256  33dcdae905f4b4267b4dc1f3efb032d79705ca8d2122e17efdecfd8162067082
+                    sha256  33dcdae905f4b4267b4dc1f3efb032d79705ca8d2122e17efdecfd8162067082 \
+                    size    47999
 
 patchfiles          patch-nbspace.diff \
                     patch-ptrdiffwarn.diff \
-                    patch-spelling.diff
-
-use_configure       no
+                    patch-spelling.diff \
+                    patch-protoMakefile.diff
 
 default_variants    +i18n
 variant i18n description adds support for multibyte characters {
     patchfiles-append   patch-i18n.diff
 }
 
-build.args          -f protoMakefile \
-                    CC="${configure.cc} \
-                        -c ${configure.optflags} [get_canonical_archflags cc]" \
-                    LINK1="${configure.cc} [get_canonical_archflags ld]"
+makefile.override   CC CFLAGS LDFLAGS
+
 build.target
 
+post-patch {
+    file copy "${worksrcpath}/protoMakefile" "${worksrcpath}/Makefile"
+}
+
 destroot {
     xinstall "${worksrcpath}/${name}" "${destroot}${prefix}/bin/"
     xinstall -m 444 "${worksrcpath}/${name}.1" "${destroot}${prefix}/share/man/man1/"
 }
+
+livecheck.type      regex
+livecheck.regex     "Par-(\\d+(?:\\.\\d+)(?:\\.\\d+)).tar.gz"
```

You can save yourself a lot of escaping here by using curly braces instead of quotation marks, since you aren't using variable substitution. (And if you were using variable substitution, then the quotation marks would be superfluous, as they are in many other places in this Portfile.)

Unless there's something special about the way this project does version numbers, I would think there isn't a reason to limit livecheck to just version numbers with one, two, or three components. Removing that limit simplifies the regex too.

```suggestion
livecheck.regex     {Par-(\d+(?:\.\d+)*)\.tar\.gz}
```

---

Ah, that's great! Thanks for that. I've added in 49aaeb5